app/controllers/candidate_interface/apply_from_find_controller.rb

```diff
@@ -81,5 +83,13 @@ def apply_on_ucas_or_apply_params
       params.require(:candidate_interface_apply_on_ucas_or_apply_form)
         .permit(:service, :provider_code, :course_code)
     end
+
+    def candidate_has_application_in_different_recruitment_cycle_year?
+      return false if @course.blank?
+      return false if current_candidate.blank?
+      return false if current_candidate.current_application.blank?
```

Comment:
Will it ever be blank? The method creates one if it's not present

Reply:
@duncanjbrown no you're right it can't be blank, will remove this line.

Reply:
Removed/amended https://github.com/DFE-Digital/apply-for-teacher-training/pull/4219/commits/b64223e733f91b9d93fcd02677e31c056e835954